.travis.yml

```diff
@@ -1,18 +1,55 @@
+sudo: required
```

Comment:
We're probably better off running Crossdock for every version, unless there is a big time savings.